src/main/java/zeenea/connector/inventory/InventoryConnection.java

```diff
@@ -0,0 +1,18 @@
+package zeenea.connector.inventory;
+
+import java.util.stream.Stream;
+import zeenea.connector.Connection;
+import zeenea.connector.common.ItemIdentifier;
+import zeenea.connector.synchronize.SynchronizeResult;
+
+/**
+ * Base interface for Inventory Connectors.
+ *
+ * @since 2.0.0
+ */
+public interface InventoryConnection extends Connection {
+
+  InventoryResult inventory();
+
+  SynchronizeResult extractItems(Stream<ItemIdentifier> items);
```

Comment:
Si on est assurés qu'un inventaire n'a besoin de retourner qu'un ensemble de "inventory items" et qu'un import ne peut retourner qu'un ensemble de "source items", je propose de simplifier comme suit : 
```suggestion
  Stream<ItemInventory> inventory();

  Stream<SourceItem> importItems(Stream<ItemIdentifier> items);
```

Inconvénient : On s'interdit d'étendre le type de retour (pour ajouter des méta-données en plus des items, par exemple)
Avantage : On simplifie la signature autant que possible